### PR TITLE
maintain dependencies

### PR DESCRIPTION
**Pull request by faddat**
Over time, dependencies can go out of date, this fixes several.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -155,10 +155,13 @@ require (
 
 replace (
 	github.com/CosmWasm/wasmd v0.31.0 => github.com/neutron-org/wasmd v0.31.1-neutron-fixes.0.20230426103416-67da724a1eaf
-	github.com/btcsuite/btcd => github.com/btcsuite/btcd v0.22.2
 	github.com/cosmos/admin-module => github.com/Ethernal-Tech/admin-module v0.0.0-20221102105340-e693f4d379c3
-	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.45.15-ics
+	// use cosmos-sdk v0.45.15-ics
+	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.45.16-ics
```
Comment (foxpy) [inline go.mod]: this comment is clearly outdated and it makes zero sense here
```suggestion
	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.45.16-ics
```

**Review by foxpy — CHANGES_REQUESTED** [review]

**Review by foxpy — APPROVED** [review]
I guess we can merge this safely right now

https://github.com/neutron-org/neutron-tests/actions/runs/5277720925

**Comment by faddat** [comment]
Yep, should be fine.  Afaik, main doesn't represent the current releases.

**Comment by pr0n00gler** [comment]
Thanks for your contribution! 
But since the PR is pretty outdated, there a couple of conflicts (some dependencies were already updated and the rest will be updated in https://github.com/neutron-org/neutron/pull/268), i'll close it

**Pull request closed without merging**